SAPex/Data/AppDbContext.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SAPex
+{
```

Comment:
wrong namespace. in your case should be SAPex.Data. but I  think we should extract it into another project